### PR TITLE
Register RData

### PR DESCRIPTION
**Pull request by alyst**
As discussed in JuliaStats/DataFrames.jl#955, `RData` is the package to read R data files.

### CODE REVIEW

**Comment by tkelman** [comment]
Were you the original author of all the code imported from DataFrames? Either this should use or acknowledge DataFrames' MIT license, or anyone who contributed to the code while it was there would have to grant permission for it to be relicensed to BSD.

File: RData/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.4
+DataFrames
+DataArrays
+FileIO
+GZip
+Compat
```
Comment (tkelman) [inline RData/versions/0.0.1/requires]: best to list minimum version requirements for the features you're using

**Comment by alyst** [comment]
@tkelman Oh, thanks, I've completely overlooked the autogenerated LICENSE. Updated to MIT and referenced the main authors.

Versions of the required packages are now specified as well.

**Comment by nalimilan** [comment]
I would have been good to preserve history when importing source from DataFrames. I guess it's still possible to do that, rebasing all commits on top of that won't create any conflicts (though you'll have to force-push to master).

**Comment by alyst** [comment]
@nalimilan Yes, that would nice. Do you know how I should call `git filter-branch` (or, perhaps, some other tool) to keep only `src/RDA.jl` and `test/RDA.jl`?

**Comment by tkelman** [comment]
you wouldn't have to force push, you could do a merge commit to combine the histories

**Comment by nalimilan** [comment]
@alyst I've never done that, but I could find several blog posts giving instructions. For example: http://gbayer.com/development/moving-files-from-one-git-repository-to-another-preserving-history/

**Pull request merged**